app/models/order_cycle.rb

```diff
@@ -63,7 +63,7 @@ class OrderCycle < ActiveRecord::Base
     if user.has_spree_role?('admin')
       where(nil)
     else
-      where(coordinator_id: user.enterprises)
+      where(coordinator_id: user.enterprises.to_sql)
```

Comment:
nice finding!!!
to_sql doesnt look good to me... I tried to_a and it works as well, do you agree to_a is better?

Reply:
I think in this case `#to_a` will trigger another query, but `#to_sql` will just convert what's there into plain sql that's perfectly usable as part of a subquery...?

Reply:
ok, that makes sense :+1:

Reply:
this is breaking 10 specs in spec/controllers/admin/schedules_controller_spec.rb
but .to_a works like a charm :heart: 
I think it's a problem with the param $1 (user_id) if you use to_sql